x230/README.md

```diff
@@ -184,12 +186,19 @@ based supply for a second USB port (like [this](https://de.aliexpress.com/item/1
 
 
 #### ifd unlock and me_cleaner: the 8MB chip
-The [Intel Management Engine](https://en.wikipedia.org/wiki/Intel_Management_Engine)
-resides on the 8MB chip (at the bottom, closer to you).
-We don't need to touch it for coreboot-upgrades in the future, but to
-enable internal flashing, we need to unlock it once, and remove the Management
-Engine for
-[security reasons](https://en.wikipedia.org/wiki/Intel_Management_Engine#Security_vulnerabilities):
+
+Flashing the bottom chip (closer to you) is optional. It has the same pinout than
+the upper chip.
+This allows you to enable in-system flashing (flashing without opening the Thinkpad)
+and/or to neuter the the 
+[Intel Management Engine](https://en.wikipedia.org/wiki/Intel_Management_Engine)
+for [security reasons](https://en.wikipedia.org/wiki/Intel_Management_Engine#Security_vulnerabilities).
+
+Choose wisely, by unlocking any software can re-flash the BIOS. By neutering the
+Intel ME you can not have a e.g.
+[measured boot with Qubes](https://github.com/QubesOS/qubes-antievilmaid/tree/master/anti-evil-maid).
```

Comment:
Could you pls eloborate a bit more why neutering the ME conflicts with `measured boot with Qubes` ?

Reply:
thanks, good question. @tomschlenkhoff do you have details?

Reply:
Sure; it's documented here:
https://github.com/QubesOS/qubes-antievilmaid/tree/master/anti-evil-maid
"In short, AEM relies on TPM and a feature found in Intel's vPro CPUs (TXT) to
detect tampering of various boot components."

Reply:
ok. but we need a refernce that the TXT module we have in Ivy Bridge mobile really doesn't work anymore after me_cleaner.

if we don't find such tests, we should not say that in our readme. people should test themselves.

Reply:
@tomschlenkhoff Thx for the explanation, however this still doesn't help me understand what this has todo with ME neutering ... ?

Reply:
I thought about it, and maybe we shouldn't mention "Qubes" here (the first and only time). Our project has nothing to do with Qubes OS and we shouldn't confuse people. Also, "measured boot" using intel's TXT is only one of more ways to have trusted boot.

Reply:
It is stated in the me_cleaner docs that it also removes the Trusted execution environment, which I assume is used interchangeably with TXT:
"Since the TXE firmware is very similar to the ME one, I adapted me_cleaner to work on it."
https://github.com/corna/me_cleaner/wiki/How-does-it-work%3F
Also see me_cleaner source, where ressources are dedicated to removing TXE.